aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -1096,6 +1096,119 @@ Tensor clone(const Tensor& src, c10::optional<c10::MemoryFormat> optional_memory
   return self;
 }
 
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ covariations ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tensor cov(const Tensor& self, 
+           const c10::optional<Tensor>& otherOpt,
+           c10::optional<bool> rowvarOpt, 
+           c10::optional<bool> biasOpt, 
+           c10::optional<int64_t> ddofOpt,
+           const c10::optional<Tensor>& fweightsOpt, 
+           const c10::optional<Tensor>& aweightsOpt) {
+  
+  const Tensor other = otherOpt.value_or(Tensor());
+  const Tensor fweights = fweightsOpt.value_or(Tensor());
+  const Tensor aweights = aweightsOpt.value_or(Tensor());
+  auto bias = biasOpt.value_or(false);
+  auto ddof = ddofOpt.value_or(0);
+  auto rowvar = rowvarOpt.value_or(true);
+
+  if(self.size(0) == 0)
```

Comment:
Egyptian-style curly braces for all if and for blocks or the linter will complain

Also a space between `if` and the opening paren